CONTRIBUTING.md

```diff
@@ -94,3 +94,72 @@ Use the following types for your commit messages:
 
 By following these guidelines, you help maintain the quality and readability of the EigenLayer codebase. We appreciate your contributions and look forward to collaborating with you!
 
+## Building and Running Tests
+
+This repository uses Foundry. See the [Foundry docs](https://book.getfoundry.sh/) for more info on installation and usage. If you already have foundry, you can build this project and run tests with these commands:
+
+```
+foundryup
+
+forge build
+forge test
+```
+
+### Contributor Setup
+
+To set up this repo for the first time, run:
+
+```bash
+make deps
+```
+
+This will:
+* Install the pre-commit hook
+* Install foundry and its tools
+* Install abigen
+
+### Running Fork Tests
+
+We have a few fork tests against ETH mainnet. Passing these requires the environment variable `RPC_MAINNET` to be set. See `.env.example` for an example. Once you've set up your environment, `forge test` should show these fork tests passing.
+
+Additionally, to run all tests in a forked environment, [install yq](https://mikefarah.gitbook.io/yq/v/v3.x/). Then, set up your environment by running the following command.
+
+`source bin/source-env.sh [goerli|local]`
```

Comment:
there's only holesky, no goerli anymore
i had a PR which changed it, but it's never merged https://github.com/Layr-Labs/eigenlayer-contracts/pull/764